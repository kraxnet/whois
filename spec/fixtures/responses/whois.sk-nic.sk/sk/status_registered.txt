% For more information on Whois status codes, please visit https://icann.org/epp
%
% This WHOIS service is provided by SK-NIC and only contains
% information pertaining to Internet domain names registered by our
% our customers. By using this service you are agreeing (1) not to use any
% information presented here for any purpose other than determining
% ownership of domain names, (2) not to store or reproduce this data in
% any way without SK-NIC's approval, (3) not to use any high-volume, automated, electronic processes
% to obtain data from this service. Abuse of this service is monitored and
% actions in contravention of these terms will result in being permanently
% blacklisted.
%
% Access to the whois service is rate limited.

Domain:                       face.sk
Registrant:                   MILA-1427-1465
Admin Contact:                SK-NIC
Tech Contact:                 ELBIA
Registrar:                    ELBI-0002
Created:                      2016-02-11
Updated:                      2017-09-07
Valid Until:                  2018-02-11
Nameserver:                   ns1.parkingcrew.net
Nameserver:                   ns2.parkingcrew.net
EPP Status:                   ok

Registrar:                    ELBI-0002
Organization:                 ELBIA, s. r. o.
Organization ID:              36702897
Phone:                        +421.904684143
Email:                        sknic@elbiahosting.sk
Street:                       Stoličková 870/4
City:                         Banská Bystrica
Postal Code:                  97401
Country Code:                 SK
Created:                      2017-09-01
Updated:                      2017-09-08

Contact:                      MILA-1427-1465
Name:                         Milan Terem
Registrar:                    ELBI-0002
Created:                      2017-09-01
Updated:                      2017-09-01

Registrar:                    SK-NIC
Organization:                 SK-NIC, a.s.
Organization ID:              35698446
Phone:                        +421.235035030
Email:                        hostmaster@sk-nic.sk
Street:                       Borská 6
City:                         Bratislava
Postal Code:                  84104
Country Code:                 SK
Created:                      2017-09-01
Updated:                      2017-09-08

Contact:                      ELBIA
Organization:                 ELBIA, s. r. o.
Name:                         Jozef Sudolský
Street:                       Stoličková 4
City:                         Banská Bystrica
Postal Code:                  97401
Country Code:                 SK
Registrar:                    ELBI-0002
Created:                      2017-09-02
Updated:                      2017-09-03
