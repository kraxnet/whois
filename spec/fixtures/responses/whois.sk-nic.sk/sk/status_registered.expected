#status
  %s == :registered

#available?
  %s == false

#response_throttled?
  %s == false

#registered?
  %s == true

#domain
  %s == "face.sk"

#domain_id
  %s raise_error(Whois::AttributeNotSupported)

#created_on
  %s %CLASS{time}
  %s %TIME{2016-02-11}

#updated_on
  %s %CLASS{time}
  %s %TIME{2017-09-07}

#expires_on
  %s %CLASS{time}
  %s %TIME{2018-02-11}

#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.parkingcrew.net"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.parkingcrew.net"

#registrar
  %s %CLASS{contact}
  %s.type         == Whois::Record::Contact::TYPE_REGISTRAR
  %s.id           == "ELBI-0002"
  %s.organization == "ELBIA, s. r. o."
  %s.email        == "sknic@elbiahosting.sk"
  %s.phone        == "+421.904684143"
  %s.address      == "Stoličková 870/4"
  %s.city         == "Banská Bystrica"
  %s.zip          == "97401"
  %s.country_code == "SK"
  %s.created_on   == Time.parse('2017-09-01')
  %s.updated_on   == Time.parse('2017-09-08')

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == "MILA-1427-1465"
  %s[0].name          == "Milan Terem"
  %s[0].organization  == nil
  %s[0].address       == nil
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country_code  == nil
  %s[0].email         == nil
  %s[0].created_on    == Time.parse('2017-09-01')
  %s[0].updated_on    == Time.parse('2017-09-01')

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "SK-NIC"
  %s[0].name          == nil
  %s[0].organization  == "SK-NIC, a.s."
  %s[0].address       == "Borská 6"
  %s[0].city          == "Bratislava"
  %s[0].zip           == "84104"
  %s[0].state         == nil
  %s[0].country_code  == "SK"
  %s[0].phone         == "+421.235035030"
  %s[0].fax           == nil
  %s[0].email         == "hostmaster@sk-nic.sk"
  %s[0].created_on    == Time.parse("2017-09-01")
  %s[0].updated_on    == Time.parse("2017-09-08")

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == "ELBIA"
  %s[0].name          == "Jozef Sudolský"
  %s[0].organization  == "ELBIA, s. r. o."
  %s[0].address       == "Stoličková 4"
  %s[0].city          == "Banská Bystrica"
  %s[0].zip           == "97401"
  %s[0].state         == nil
  %s[0].country_code  == "SK"
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == nil
  %s[0].created_on    == Time.parse("2017-09-02")
  %s[0].updated_on    == Time.parse("2017-09-03")
